core/templates/dev/head/css/oppia.css

```diff
@@ -1189,10 +1189,27 @@ pre.oppia-pre-wrapped-text {
   position: relative;
 }
 
-.oppia-editable-section:hover .oppia-state-content-display,
-.oppia-editable-section:hover .oppia-rule-preview {
```

Comment:
Can you please tell me where else it is used ?
I am not sure why line 1193 was needed, and I thik that line 1192 does the masking job.